oops/meshgrid.py

```diff
@@ -227,20 +337,104 @@ def uv_w_derivs(self, time=None):
     def duv_dlos(self, time=None):
         return self.uv_w_derivs(time).d_dlos
 
-################################################################################
-# UNIT TESTS
-################################################################################
+    ############################################################################
+    # Center methods
+    ############################################################################
+
+    def center_los_w_derivs(self, time=None):
+
+        # Return from internal dictionary if present
+        try:
+            return self.filled_center_los_w_derivs[time]
+        except (KeyError, TypeError):
+            pass
+
+        # Evaluate the LOS anew
+        los_ = self.fov.los_from_uvt(self.center_uv_w_duv_duv, time=time,
```

Comment:
TMI: Because "los" is a method name, and because los_w_derivs is highly parallel to method los, which also uses a variable called "los_". I changed all occurrences to "result".